src/main/java/seedu/apollo/ui/Ui.java

```diff
@@ -270,6 +270,67 @@ public void printModuleList(ModuleList allModules) {
         printTotalModularCredits(allModules);
     }
 
+    /**
+     * Prints out the timetable of a module added by the user in the module list.
+     *
+     * @param newModule Module that has just been added.
+     */
+    public void printModuleListWithLesson(Module newModule, ArrayList<Timetable> timetableList) {
+        System.out.println("These are your classes for Module " + newModule.getCode() + ": \n");
+        for (Timetable timetable : timetableList) {
+            System.out.println(timetable.getLessonType() + " " + timetable.getClassnumber() + '\n' +
+                    "   " + timetable.getDay() + " " + timetable.getStartTime() + " - " +
+                    timetable.getEndTime());
```

Comment:
I think like what one of the PED comments said, maybe it would be better for us to format it so that start time and end time comes before the class itself? otherwise you can just use the default toString for timetable i think it prints in this format liao no need to so mafan get this get that HAHAHAH

Reply:
ok update i went to see the toString doesn't actually print all of the data here, another idea is to create a method inside Timetable itself (eg. public String fullTimetable) which returns all the stuff here in a String

Reply:
> ok update i went to see the toString doesn't actually print all of the data here, another idea is to create a method inside Timetable itself (eg. public String getFullTimetable) which returns all the stuff here in a String

This is possible, let me look into it.